students/slupek_bartosz/lesson_03_functions/collatz.py

```diff
@@ -0,0 +1,15 @@
+def collatz(number):
+    if number % 2 == 0:
+        print(number // 2)
+        return number // 2
+    elif number % 2 == 1:
```

Comment:
ten warunek jest zbędny, ponieważ działanie `liczba % 2` zwróci nam 0 (parzysta)  lub 1 (nie parzysta). Czyli skoro ` if number % 2 == 0:` zostało niespełnione to znaczy, że dostaliśmy w wyniku jedynkę i nie musimy tego sprawdzać.